tensorflow_addons/layers/normalizations.py

```diff
@@ -25,6 +25,9 @@
 @tf.keras.utils.register_keras_serializable(package="Addons")
 class GroupNormalization(tf.keras.layers.Layer):
     """Group normalization layer.
+    
+    Source: "Group Normalization" (Yuxin Wu & Kaiming He, 2018)
```

Comment:
Reference is [mentioned](https://github.com/tensorflow/addons/blob/174fc18ed03d9a30ddbbe2a9d6c3087a4766973d/tensorflow_addons/layers/normalizations.py#L74) at the end.

Reply:
Could you modify it such that only one exists. It might be redundant otherwise.

Reply:
My bad. Didn't notice it since it was out of my viewport :D

Reply:
Ill change it again and put it at the top. Leaving it at the bottom seems to be inconsistent with the way this was done in other papers.